src/accordion/accordion-section.svelte

```diff
@@ -0,0 +1,37 @@
+<script>
+  import Button from '../button/button.svelte';
+  import { createEventDispatcher } from 'svelte';
+  import classes from '../_utils/classes.js';
+
+  let _class = null;
+  export { _class as class };
+
+  export let label = null;
+  export let open = false;
+
+  const selfControl = {
+    close() {
+      open = false;
+    },
+    toggle() {
+      if (!open) {
+        dispatch('panel-open', selfControl);
```

Comment:
Why not dispatching when it is closed as well? Also, where is the native event?

Reply:
Technically, this is an internal event that wasn't really meant to be listened on, but now that I think about it, why not. Where's the native event, you ask? There might not be. We don't know what has triggered the toggle of the panel, it's just a callback that a user calls at their liking